### PR TITLE
news/blog: Initial submission of RocksDB Tuning Deep Dive

### PR DESCRIPTION
**Pull request by markhpc**
Initial submission of the RocksDB Tuning Deep Dive Article.

Signed-off-by: Mark Nelson <mnelson@redhat.com>

### CODE REVIEW

**Comment by markhpc** [comment]
Looks like there's some issue with side-by-side images.  Probably could clean up some other things to look a bit better too.

File: src/en/news/blog/2022/rocksdb-tuning-deep-dive/index.md [hunk 1]
```diff
@@ -0,0 +1,301 @@
+---
+title: Ceph RocksDB Tuning Deep-Dive
```
Comment (ljflores) [inline src/en/news/blog/2022/rocksdb-tuning-deep-dive/index.md:2]: You could prexif this with "Quincy @ Scale" to match @pcuzner's.
```suggestion
title: Quincy @ Scale: Ceph RocksDB Tuning Deep-Dive
```

File: src/en/news/blog/2022/rocksdb-tuning-deep-dive/index.md [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+date: 2022-07-14
+image: images/octopus.jpg
```
Comment (ljflores) [inline src/en/news/blog/2022/rocksdb-tuning-deep-dive/index.md:4]: Love the image XD

Comment (markhpc) [reply src/en/news/blog/2022/rocksdb-tuning-deep-dive/index.md:4]: Thanks!

**Review by Thingee — APPROVED** [review]

**Pull request merged**